webapp/src/components/MeetingAccordion.tsx

```diff
@@ -21,26 +22,42 @@ interface MeetingAccordionProps {
 const MeetingAccordion = ({ meeting }: MeetingAccordionProps) => {
   return (
     <Accordion>
-      <AccordionSummary expandIcon={<ExpandMoreIcon />}>
-        <ListItem>
-          <MuiLink
-            component={ReactRouterLink}
-            to={`/meetings/${meeting.id}`}
-            underline="none"
-            sx={{ color: 'text.primary' }}
-            onClick={e => {
-              e.stopPropagation();
+      <AccordionSummary
+        expandIcon={
+          <IconButton>
```

Comment:
I'm curious what this `IconButton` does. The examples in the docs don't have it.

https://mui.com/components/accordion/

Reply:
The `IconButton` creates a grey outline when hovering over the `ExpandMoreIcon`.  I noticed the grey outline in the product spec.  Should it be removed?

Reply:
The product spec uses an entirely different icon, and a different approach to revealing the detailed content. I recall we decided to use a different icon from the spec so that it matched more closely with user expectations. I believe that the example in the MUI docs matches with user expectations, so it would make sense to follow that example.

Reply:
Yes, this also confuses the user as to what to click to expand/collapse the accordion.  I'll remove it.